src/codegeneration/InstantiateModuleTransformer.js

```diff
@@ -118,6 +118,25 @@ class DeclarationExtractionTransformer extends HoistVariablesTransformer {
 }
 
 /**
+ * Replaces __moduleName with $__context.id
```

Comment:
I would find it helpful if the reason -- back compat I assume -- was included in the comment.